util/keyfile/src/lib.rs

```diff
@@ -37,7 +37,7 @@ pub fn read_keyfile_data<R: std::io::Read>(buffer: &mut R) -> Result<RootIdentit
 
 /// Write user public address to disk
 pub fn write_pubfile<P: AsRef<Path>>(path: P, addr: &PublicAddress) -> Result<(), std::io::Error> {
-    File::create(path)?.write_all(&serde_json::to_vec(&addr)?)?;
+    File::create(path)?.write_all(&mc_util_serial::encode(addr))?;
```

Comment:
Maybe this breaks some Python code somewhere?

Reply:
hmm... okay, lets talk to @rjwalters